### PR TITLE
Get rid of double negation in Events semantic conventions

### PR DESCRIPTION
**Pull request by martinknechtel**
Is that what you intended?

### CODE REVIEW

**Comment by martinknechtel** [comment]
@scheler FYI

File: specification/logs/semantic_conventions/events.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ possible that the existing system does not have the equivalent of a name or
 requires multiple fields to identify the structure of the events. In such cases,
 OpenTelemetry recommends using a combination of one or more fields as the name
 such that the name identifies the event structurally. It is also recommended that
-the event names have low-cardinality, so care must be taken to not use fields
-that identify the instance of the Event and not the class of Events.
+the event names have low-cardinality, so care must be taken to use fields
+to identify the class of Events but not the instance of the Event.
```
Comment (yurishkuro) [inline specification/logs/semantic_conventions/events.md:27]:
```suggestion
that identify the class of Events but not the instance of the Event.
```

**Review by scheler — APPROVED** [review]
@martinknechtel thanks for catching the error.

**Comment by MrAlias** [comment]
@martinknechtel please sign the the CLA so these changes can be considered.

**Review by arminru — APPROVED** [review]

**Comment by martinknechtel** [comment]
@scheler @MrAlias done, can be merged from my perspective

**Review by tigrannajaryan — APPROVED** [review]

**Review by yurishkuro — APPROVED** [review]

**Pull request merged**